src/widgets/types.ts

```diff
@@ -6,8 +6,8 @@ export interface ISlackWidget {
 }
 
 export enum SlackWidgetDataMode {
-  PER_INSTALLER = "per_installer",
-  PER_USER = "per_user",
+  PER_INSTALLER = "PER_INSTALLER",
+  PER_USER = "PER_USER",
```

Comment:
just updated these to the caps version if it's all the same to you, some slight convenience on my end and matches our API conventions more closely